### DISCUSSION TITLE
What's about the reactions stuff?

### DISCUSSION THREAD

**Discussion by albotroz** [usememos/memos#4106] [Q&A] [answered]
I have installed memos on my proxmox server in a lxc-container as a test and played around. I noticed that nothing appears when I click on the smiley button. So no selection of smileys.
Under Settings -> Memos I see a field with “Reactions” and it says “Input”.
Does this have anything to do with it? In any case, I can't change anything else in the “Memos” area because the error message “Reactions must not be empty.” appears.

Very confusing...

**Comment by RoccoSmit** [comment, marked as answer]
Are you upgrading or starting a fresh instance?
Pr https://github.com/usememos/memos/pull/4094 was committed yesterday to resolve the “Reactions must not be empty.” error message for fresh installs

**Reply by albotroz** [reply]
> Are you upgrading or starting a fresh instance? Pr #4094 was committed yesterday to resolve the “Reactions must not be empty.” error message for fresh installs

Thanks for reply. New instance. However. I just noticed by chance that if you go to the “Input” section, put for example :-) in the “Input” area, then you can save the other settings. And use the smiley :-) on the side.

**Reply by RoccoSmit** [reply]
Have a look at this comment for a list of default reactions to add https://github.com/usememos/memos/issues/4060#issuecomment-2439878664
